packages/celotool/src/cmds/deploy/upgrade/verify-contracts.ts

```diff
@@ -0,0 +1,47 @@
+import * as yargs from 'yargs'
+import { InitialArgv } from '../../deploy/initial'
+import { switchToClusterFromEnv } from 'src/lib/cluster'
+import { getBlockscoutUrl } from 'src/lib/endpoints'
+import { execCmd } from 'src/lib/utils'
+
+export const command = 'verify-contracts'
+
+export const describe = 'verify the celo smart contracts in blockscout'
```

Comment:
How is this command different than the initial one?

Reply:
maybe better to use have only one command `verify-contracts` and use the `contract` arg as either:
- a single contract address
- a list of contract addresses
- all contracts

Reply:
No difference at all. Really it does not makes sense to have an `upgrade` command for this, so I will remove. If we want to "rerun" the verification process we can use `initial` again, and the contracts already verified will be skipped.
I have not added a `destroy` command to this. It is possible to clean the blockscout database with the contract code if somebody finds this useful, but I do not find very useful to unverify a contract, but if you see otherwise we can add.